l10n_it_fatturapa_out/migrations/16.0.1.0.0/pre-migrate.py

```diff
@@ -20,11 +20,15 @@ def migrate(env, version):
         (table_name, "%s" % constraint_name),
     )
 
-    if env.cr.fetchall()[0]:
+    res = env.cr.fetchall()
+    if res and res[0]:
         alter_table_sql = sql.SQL("ALTER TABLE {} DROP CONSTRAINT {}").format(
             sql.Identifier(table_name), sql.Identifier(constraint_name)
         )
         drop_line_sql = "DELETE FROM ir_model_constraint WHERE NAME = %s"
 
         openupgrade.logged_query(env.cr, alter_table_sql)
         openupgrade.logged_query(env.cr, drop_line_sql, [constraint_name])
+
+    view = env.ref("l10n_it_fatturapa_out.view_fatturapa_out_attachment_form")
+    view.inherit_id = False
```

Comment:
Questo si può rimuovere? Per i motivi di cui parlammo due settimane fa in https://github.com/OCA/l10n-italy/pull/3694#pullrequestreview-1832016724

Reply:
non mi ricordavo più...